apps/mobile/src/screens/tasks/tasks.tsx

```diff
@@ -1,9 +1,20 @@
 import React from "react";
 
-import { ScreenWrapper, Tag, Text } from "~/libs/components/components";
+import {
+	ScreenWrapper,
+	SliderContent,
+	Tag,
+	Text,
+} from "~/libs/components/components";
 import { useAppDispatch, useEffect } from "~/libs/hooks/hooks";
+import { type colorToGradientColors } from "~/libs/maps/maps";
 import { actions as userActions } from "~/slices/users/users";
 
+const sliderData: {
+	color: keyof typeof colorToGradientColors;
+	label: string;
+}[] = [{ color: "red", label: "Love" }];
```

Comment:
Lets move 
```
{
	color: keyof typeof colorToGradientColors;
	label: string;
}
```
to a separate type SliderData. It will make code easier to read